src/ex1/style.css

```diff
@@ -0,0 +1,203 @@
+body {
+  background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+  /* background: linear-gradient(90deg, red, yellow, green); */
+  background-size: 400% 400%;
+  animation: gradient 15s ease infinite;
+  height: 100vh;
+
+  display: flex;
+  flex-direction: center;
+  justify-content: center;
+  align-items: baseline;
+  overflow: hidden;
+}
+
+@keyframes gradient {
+  0% {
+    background-position: 0% 50%;
+  }
+  50% {
+    background-position: 100% 50%;
+  }
+  100% {
+    background-position: 0% 50%;
+  }
+}
+
+@font-face {
+  font-family: Poppins;
+  src: url("../assets/fontStyle/Poppins.ttf");
+}
+div {
+  font-family: Poppins;
+}
+
+.container {
+  display: flex;
+  margin-top: 20vw;
+  align-items: center;
+  border-radius: 5px;
+  background-color: white;
+
+  box-shadow: 1px 1px 5px black;
+}
+
+.top_bar_container {
+  display: flex;
+  justify-content: row;
+  margin-bottom: 15px;
+}
+.buttom_bar_container {
+  margin-top: 18px;
+  height: 30px;
+  /* border: 2px solid; */
+  width: 100%;
+  margin-bottom: 5px;
+  justify-content: space-between;
+  align-items: center;
+  display: none;
+}
+
+.task {
+  background-color: rgb(235, 232, 232);
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 30px;
+  padding-left: 5px;
+  font-family: "Times New Roman", Times, serif;
+  margin: 0;
+
+  border-radius: 1px;
+}
+
+.task_container {
+  display: flex;
+  justify-content: row;
+  align-items: center;
+  /* border: 1px solid;
+  border-color: #ffc0cb; */
+  width: 100%;
+  margin-bottom: 4px;
+  transition: 5s;
+}
+
+.task_container:focus-within .task {
+  border: 1px solid;
+  border-color: #ffc0cb;
+  transition: step-start;
+}
+
+.task_container:hover .delete_task_button {
+  display: flex;
+  transition: step-start;
+  cursor: pointer;
+}
+.task_container:hover .task {
```

Comment:
according to the instructions the item itself should change color and cursor when hovered
"Change the background and cursor type of an item when hovered on"